.github/workflows/testing.yml

```diff
@@ -22,9 +22,17 @@ jobs:
       run: go build -v ./...
 
     - name: Test
-      env: 
+      env:
+        FIVETRAN_API_URL : "https://api-staging.fivetran.com/v1"
         FIVETRAN_APIKEY : ${{ secrets.FIVETRAN_APIKEY }}
         FIVETRAN_APISECRET : ${{ secrets.FIVETRAN_APISECRET }}
         FIVETRAN_TEST_CERTIFICATE_HASH : ${{ secrets.FIVETRAN_TEST_CERTIFICATE_HASH }}
         FIVETRAN_TEST_CERTIFICATE : ${{ secrets.FIVETRAN_TEST_CERTIFICATE }}
+        FIVETRAN_GROUP_ID : "sepulchre_settlement"
+        FIVETRAN_GROUP_NAME: "GoSdkTesting"
+        FIVETRAN_USER_ID : "recoup_befell"
+        FIVETRAN_USER_EMAIL : "dev-markov+go-fivetran-sdk@fivetran.com"
+        FIVETRAN_USER_GIVEN_NAME : "Go"
+        FIVETRAN_USER_FAMILY_NAME : "5Tran"
+        FIVETRAN_USER_PHONE : "+1234567890"
```

Comment:
I think we can verify user and group just by IDs and there's no need to store all other details.

Reply:
All other details is a part of test. Test should not depend on environment values ideally. 
I think we should refactor tests rather than use service user/group to perform tests on it.

It's better to create group and user first and then test get endpoints. 

Anyway - now it's better to leave here only identification-related values (IDs)